cincoctrl/cincoctrl/templates/users/user_form.html

```diff
@@ -7,9 +7,7 @@
 {% endblock title %}
 {% block content %}
   <h1>Change Password</h1>
-  <form class="form-horizontal"
-        method="post"
-        action="{% url 'users:detail' user.pk %}">
+  <form class="form-horizontal" method="post" action=".">
```

Comment:
What does the `.` in `action="."` do? I've never seen that before

Reply:
ah, we probably don't need the action at all.  I was fixing a stupid mistake in this form where the form was actually submitting to a completely different view and thus bypassing the actual action it was supposed to take.